src/rest_framework_jwt/blacklist/views.py

```diff
@@ -0,0 +1,37 @@
+from rest_framework import status
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.viewsets import ModelViewSet
+
+from rest_framework_jwt.blacklist.models import BlacklistedToken
+from .serializers import BlacklistTokenSerializer
+from ..authentication import JSONWebTokenAuthentication
+
+
+class BlacklistView(ModelViewSet):
+    permission_classes = (IsAuthenticated, )
+    authentication_classes = (JSONWebTokenAuthentication, )
```

Comment:
The [documentation](https://github.com/Styria-Digital/django-rest-framework-jwt/blob/master/docs/index.md#usage) states the authentication class should be defined in the `REST_FRAMEWORK` settings to make it globally available. This renders this assignment redundant at best, and difficult to debug at worst (if someone uses a customized `JSONWebTokenAuthentication` class).